### PR TITLE
Update version and release information.

### PR DESCRIPTION
**Pull request by JamesTheZ**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+## CPU end-to-end optimization
+
+Support end-to-end optimization for X86 and AArch64 CPUs.
+
+## TorchBlade/TensorFlowBlade clustering and optimizing with TensorRT
+
+According to the supported operators of TensorRT, cluster sub-graphs and apply TensorRT optimization for both TensorFlow and PyTorch models.
+
+## Accelerating PyTorch Training
```
Comment (qiuxiafei) [inline RELEASE.md:46]: Shall we announce this? @Yancey1989 @fortianyou

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+Enhance the CI process of BladeDISC repo, which helps the people from community to contribute to BladeDISC more conveniently and efficiently.
+
+## TorchBlade bazel build
+Migrate TorchBlade's compilation toolchain from the original CMake to bazel, enhancing maintainability.
+
+# Other
+
+## Benchmark preparation
+
+Prepare a set of commonly used models as the examples for BladeDISC.
```
Comment (wyzero) [inline RELEASE.md:78]: Maybe add ORT as well. I added a torch cpu demo to compare the performance of disc and ORT as well.

File: RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
+# Release 0.2.0
+
+# Performance Optimization
+
+## GPU stitch fusion
```
Comment (wyzero) [inline RELEASE.md:5]: Should we mention cpu stitch as well? I am not sure if we mentioned this feature in previous release notes.

Comment (JamesTheZ) [reply RELEASE.md:5]: CPU stitch is already released when we open-source BladeDISC project. If we want to announce features developed before open-source the release note of v0.2.0, we may need a rule to select the features from all the existing techniques.

File: RELEASE.md (hunk 2 above)
Comment (JamesTheZ) [reply RELEASE.md:78]: Done.

File: RELEASE.md (hunk 3 above)
Comment (JamesTheZ) [reply RELEASE.md:5]: @wyzero I have attached the link about the materials of CPU stitch optimizations.

File: RELEASE.md (hunk 1 above)
Comment (JamesTheZ) [reply RELEASE.md:49]: After an offline discussion, we decide to keep it as it is.

**Review by wyzero — APPROVED** [review]
LGTM

**Pull request merged**